msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-meeting-stage.md

```diff
@@ -0,0 +1,196 @@
+        "privateChatTab",
+        "meetingChatTab",
+        "meetingDetailsTab",
+        "meetingSidePanel",
+        "meetingStage"
+     ]
+    }
+  ]
+```
+
+---
+
+#### Other examples
+
+Default context for tabs (if not specified) is:  
+
+```json
+"context":[ 
+  "channelTab", 
+  "privateChatTab", 
+  "meetingChatTab", 
+  "meetingDetailsTab" 
+     ] 
+```
+
+To prevent an app from showing in non-meeting group chats, you would set the following context on their tab:
+
+```json
+"context":[ 
+  "meetingSidePanel", 
+  "meetingChatTab", 
+  "meetingDetailsTab" 
+     ] 
+```
+
+To target just the in-meeting side panel experience:  
+
+```json
+"context":[ 
+  "meetingSidePanel” 
+     ] 
+```
+
+### Advance Tab SDK APIs
+
+The Microsoft Teams JavaScript client SDK is a rich SDK used to create Tabs using Javascript. Use latest TeamsJS (V.2.0 or later) to work in Teams, Office, and Outlook. For more information, see [Teams JavaScript client SDK](/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?branch=pr-en-us-6722&tabs=javascript%2Cmanifest-teams-toolkit).
+
+### Frame context
+
+Microsoft Teams JavaScript library exposes the frameContext in which your meeting tab URL is loaded in the getContext API. The possible values of frameContext are content, task, setting, remove, sidePanel. This allows you to build customized experiences according to the size of the frame in which your app is loaded. For more information, see [getContext API](/microsoftteams/platform/tabs/how-to/access-teams-context?tabs=teamsjs-v2).
+
+## Code sample
+
+|Sample name | Description | C# | Node.js |
+|----------------|-----------------|--------------|----------------|
+| Meeting app | Demonstrates how to use the Meeting Token Generator app to request a token. The token is generated sequentially so that each participant has a fair opportunity to contribute in a meeting. The token is useful in situations like scrum meetings and Q&A sessions. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-token-app/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-token-app/nodejs) |
+|Meeting stage sample | Sample app to show a tab in meeting stage for collaboration | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-stage-view/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-stage-view/nodejs) |
+|Meeting side panel | Sample app to show how to add agenda in a meeting side panel | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-sidepanel/csharp) |-|
+| In-meeting notification | Demonstrates how to implement in-meeting notification using bot. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/nodejs) |
+
```

Comment:
Purely because I worked on it, and additional sample to list here is 

| In-Meeting Document Signing | Demonstrates how to implement a document signing Teams app. Includes sharing specific app content to stage, Teams SSO and user specific stage view. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-share-to-stage-signing/csharp) | |